tutorials/power_distribution/api.h

```diff
@@ -0,0 +1,18 @@
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+gpio_t DASHBOARD_POWER = PB0;
+gpio_t SERVICE_SECTION_POWER = PB1;
+gpio_t THROTTLE_POWER = PB2;
+
+void power_device(gpio_t device);
+// Turn on the specified device
+
+void power_all();
```

Comment:
Generally speaking, if a function doesn't take arguments, you should put `void` between the parentheses:

```c
void power_all(void);
```

Same below and in other file.